### PR TITLE
Add a linting script

### PR DESCRIPTION
**Pull request by DanielRamosAcosta**
I've created this small script so we can track down easier issues like #67

### CODE REVIEW

**Comment by dsifford** [comment]
Neat! Sorry for the delay... Looking over this now...

File: lint.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    "editor.tokenColorCustomizations"
+];
+
+const fileToLint = process.argv[2];
+
+if (!fileToLint) {
+    throw new Error("You need to specifiy the path of the theme to lint");
+}
+
+async function scrapThemeAvailableKeys() {
```
Comment (dsifford) [inline lint.js]: spelling

File: lint.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    return availableKeys;
+}
+
+async function getTheme() {
+    const fileContent = await readFile(fileToLint, "utf8");
+    const theme = JSON.parse(fileContent);
+    return theme;
+}
+
+async function main() {
```
Comment (dsifford) [inline lint.js]: Just for style points, maybe just make this an IIFE?

So, turn this...

```js
async function main() {
    // ...
}
main().catch(err => console.error(err));
```

to this...

```js
(async () => {
    // ...
})().catch(err => console.error(err));
```

File: lint.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+if (!fileToLint) {
+    throw new Error("You need to specifiy the path of the theme to lint");
+}
+
+async function scrapThemeAvailableKeys() {
+    const data = await get(THEME_COLOR_REFERENCE_URL);
+
+    const matches = data.match(new RegExp("<code>.+?</code>", "g"));
+
+    const availableKeys = [...matches]
```
Comment (dsifford) [inline lint.js]: yowza! Really muscling this one here.. Assuming this works, but might be fragile.

File: lint.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+const THEME_COLOR_REFERENCE_URL =
+    "https://code.visualstudio.com/docs/getstarted/theme-color-reference";
+
+const NOT_THEME_KEYS = [
+    "workbench.colorCustomizations",
+    "editor.tokenColorCustomizations"
+];
+
+const fileToLint = process.argv[2];
```
Comment (dsifford) [inline lint.js]: Should this go in the main function?

**Comment by DanielRamosAcosta** [comment]
I'll answer your reviews tomorrow, here is 2 AM, gotta sleep lol

**Comment by DanielRamosAcosta** [comment]
Alright! I've fixed your comments @dsifford 😄

File: lint.js (hunk 1 above)
Comment (DanielRamosAcosta) [reply lint.js]: Oh that one 🤣 thanks!

**Comment by DanielRamosAcosta** [comment]
Ok, I've fixed the function name and I've formatted the code.

I'll try create another PR for reading from the `.yml` on friday 😄

**Comment by DanielRamosAcosta** [comment]
@diegotoral I was refactoring the `build.js` script, and at [line 27](https://github.com/dracula/visual-studio-code/blob/master/build.js#L27) it seems that you are trying to remove keys with null value. But this actually won't make effect as you never use it down before.

Is this normal?

**Pull request merged**